README.md

```diff
@@ -93,7 +105,21 @@ Your Answer:
 
 What are the differences between calling and referencing a function? Please provide examples of each.
 
-```text
+```
+Calling a function is to invoke the code it contains. So if I have
+
+var myFunction = function () {
+  var instructor = "Joe";
+  console.log(instructor);
+}
+
+I can call this function with
+
+myFunction();
+
+Referencing a function is when you do something that acknowledges the existence of a function but does not run any of its code. I can reference myFunction by setting a variable that points to it like this:
+
+var myExample = myFunction;
```

Comment:
:+1:  I would say referencing acknowledges the function and does not run it _yet_